nike_project/lib/common/Button/elevated_button_custom.dart

```diff
@@ -5,32 +5,27 @@ class ElevatedButtonCustom extends StatelessWidget {
   final Widget child;
   final Color color;
   final VoidCallback? onPressed;
-  final double borderRadius;
-  final double height;
-  final double width;
 
-  const ElevatedButtonCustom(
-      {Key? key,
-      required this.child,
-      required this.width,
-      required this.color,
-      required this.onPressed,
-      required this.borderRadius,
-      required this.height})
-      : super(key: key);
+  const ElevatedButtonCustom({
+    Key? key,
+    required this.child,
+    required this.color,
+    required this.onPressed,
+  }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
+    Size size = MediaQuery.of(context).size;
     return SizedBox(
-      width: width,
-      height: height,
+      width: size.width,
+      height: size.height * 0.06,
```

Comment:
when you declare height * 0.06 then all button same height .I think you must dynamic height

Reply:
All buttons should use the same size